### DISCUSSION TITLE
Tailwind support

### DISCUSSION THREAD

**Discussion by rangelkoli** [TroyAlford/react-jsx-parser#268] [Q&A]
Is it possible to use Tailwind CSS classes in the output, because i tried but some classes are just not being rendered the way they should be

**Comment by TroyAlford** [comment]
This lib should be pretty agnostic to that. If you provide a minimal repro, I'll try to take a look -- but I don't see any reason why the parser would interfere with your class names, on the surface.

**Comment by ArchitGoyal2005** [comment]
should i add something to the bindings if i am using tailwind? bcoz i am getting the same issue
